src/common/pure/TransactionSubmittedContent/SurplusModal.tsx

```diff
@@ -10,6 +20,128 @@ import { TokenAmount } from 'common/pure/TokenAmount'
 const SELL_SURPLUS_WORD = 'got'
 const BUY_SURPLUS_WORD = 'saved'
 
+function getTwitterText(surplusAmount: string, surplusToken: string) {
+  const surplus = `${surplusAmount} ${surplusToken}`
+  return encodeURIComponent(
+    `Hey, I just ${SELL_SURPLUS_WORD} ${surplus} on @CoWSwap! 🐮💸\n\nStart swapping on swap.cow.fi`
```

Comment:
shouldn't this message receive the `order.kind` as a parameter (or directly the `surplusWord`)? so it works for buy orders too?

Reply:
Yes you are right. Just committed a fix.